modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/list/List.js

```diff
@@ -115,7 +115,7 @@ const ListItem = ({item, schema}) => {
 				<ClayList.ItemField>
 					<ImageRenderer
 						sticker={sticker && item[sticker]}
-						value={{src: item[image]}}
+						value={item[image]}
```

Comment:
This is a bugfix, right? `src: ` was wrong? I'm double-checking that we're not changing API, as that would break things in commerce.

Reply:
Previous implementation assumes that `item[image]` is a string (URI) for the `<img src />` attribute. Allowing anything as value we can cover other use cases and provide other attributes as well.